docs/src/modules/components/AppFrame.js

```diff
@@ -277,7 +277,7 @@ function AppFrame(props) {
                 {LANGUAGES_LABEL.map((language) => (
                   <MenuItem
                     component="a"
-                    data-no-link="true"
+                    data-no-markdown-link="true"
```

Comment:
What is this for? No objection, just want to know what it does.

Reply:
MarkdownLinks tries to handle all the `<a>` clicks. This was done to have smooth routing navigation on the markdown. `data-no-markdown-link` opt-out from this behavior.